modules/microshift-disabling-lvms-csi-driver.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+// * microshift_configuring/microshift-disable-lvms-csi-provider-csi-snapshot.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-lvms-csi-driver_{context}"]
+= Disabling deployments that run the CSI driver implementations
+
+Before you disable the CSI driver implementation pods, you must manually stop any user workloads that have attached volumes. Use the following procedure to disable installation of the CSI implementation pods.
+
+[NOTE]
+====
+{microshift-short} will not delete CSI driver implementation pods. You must configure {microshift-short} to disable installation of the CSI driver implementation pods during the startup process.
+====
+
+.Procedure
+
+. To disable installation of the CSI driver, enter the `driver` value under the `storage` section of the {microshift-short} configuration file:
```

Comment:
My preference is to specifically call out `/etc/microshift/config.yaml` instead of just saying MicroShift configuration file.

Reply:
@copejon This can be changed as suggested?

Reply:
Yes, good point.

Reply:
done in both "microshift-disabling-lvms-csi-driver.adoc" and "microshift-disabling-lvms-csi-snapshot.adoc"